src/main/java/org/dinosaur/foodbowl/domain/photo/application/FileSystemPhotoUtils.java

```diff
@@ -0,0 +1,89 @@
+package org.dinosaur.foodbowl.domain.photo.application;
+
+import static org.dinosaur.foodbowl.domain.photo.exception.FileExceptionType.FILE_BASE_NAME_ERROR;
+import static org.dinosaur.foodbowl.domain.photo.exception.FileExceptionType.FILE_EXTENSION_ERROR;
+import static org.dinosaur.foodbowl.domain.photo.exception.FileExceptionType.FILE_FORMAT_ERROR;
+import static org.dinosaur.foodbowl.domain.photo.exception.FileExceptionType.FILE_TRANSFER_ERROR;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import org.dinosaur.foodbowl.global.exception.FileException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+@Component
+public class FileSystemPhotoUtils implements PhotoUtils {
+
+    private static final Set<String> IMAGE_EXTENSIONS = Set.of("jpeg", "jpg", "png", "webp");
+    private static final String SLASH = "/";
+    private static final String DASH = "-";
+    private static final String DOT = ".";
+
+    @Value("${file.dir}")
+    private String rootDirectory;
```

Comment:
```suggestion
    private final String rootDirectory;
    
    public FileStstemPhotoUtils(@Value("${file.dir}") String rootDirectory) {
        this.rootDirectory = rootDirectory;
    }
```
불변으로 만드는 방법도 존재합니다~~